bundles/org.openhab.binding.velbus/README.md

```diff
@@ -2,139 +2,232 @@
+
+On Windows `port` will refer to one of the COM devices such as `COM1`, `COM2`, etc. The Device Manager system utility can be used to determine the exact COM port number to use.
+
+In a `.things` file, a USB connection to a Velbus configuration module might be configured like so:
 
 ```java
+Bridge velbus:bridge:1 [ port="/dev/serial/by-id/usb-Velleman_Projects_VMB1USB_Velbus_USB_interface-if00" ]
+// or
 Bridge velbus:bridge:1 [ port="COM1" ]
 ```
 
-For the Velbus Network Bridge it is necessary to specify the IP Address or hostname and the port of the Velbus network server.
-This will usually be either the loopback address `127.0.0.1`, and port number.
-Or the specific IP of the machine `10.0.0.110` , and port number.
+### Velbus Network Bridge
+
+For the Velbus Network Bridge it is necessary to specify the address (either an IP address or a hostname) and the port of a Velbus network server.
 
-In the things file, this might look like
+In a `.things` file, a network bridge running on the same machine at port 6000 would be configured like so:
 
 ```java
-Bridge velbus:networkbridge:1 "Velbus Network Bridge - Loopback" @ "Control" [ address="127.0.0.1", port=6000 ]
+Bridge velbus:networkbridge:1 "Velbus Network Bridge - Loopback" @ "Control" [ address="localhost", port=6000 ]
 ```
 
-Optionally, both the serial bridge and the network bridge can also update the realtime clock, date and daylight savings status of the Velbus modules.
-This is achieved by setting the Time Update Interval (in minutes) on the bridge, e.g.:
+### Realtime Clock Synchronization
+
+Optionally, the openHAB Velbus binding can synchronize the realtime clock, date and daylight savings status of the Velbus modules. This is achieved by setting the Time Update Interval (in minutes) on the bridge thing. For example:
 
 ```java
 Bridge velbus:bridge:1 [ port="COM1", timeUpdateInterval="360" ]
 ```
 
-The default time update interval is every 360 minutes.
-Setting the interval to 0 minutes or leaving it empty disables the update of the realtime clock, date and daylight savings status of the Velbus modules.
+If `timeUpdateInterval` is not specified, the time will be updated every 360 minutes by default. In order to disable this behaviour, set the interval to 0 or an empty string.
 
-In case of a connection error, the bridges can also try to reconnect automatically.
-You can specify at which interval the bridge should try to reconnect by setting the Reconnection Interval (in seconds), e.g.:
+### Reconnection
+
+In case of a connection error, a Velbus bridge will attempt to reconnect every 15 seconds by default. You can modify the bridge reconnection interval by specifying the `reconnectionInterval` parameter (in seconds):
```

Comment:
```suggestion
In case of a connection error, a Velbus bridge will attempt to reconnect every 15 seconds by default.
You can modify the bridge reconnection interval by specifying the `reconnectionInterval` parameter (in seconds):
```